grafana/grafana
# LogContext: Fix filtering out log lines with the same entry

**What is this feature?**

There was a bug where LogContext filtered out too many log lines. Basically log lines with the exact entry were filtered out. 

**How to reproduce:**
1. Make sure you are on latest main
2. Run a Loki Query: `{place="luna"} | line_format "test"`
3. Open Log Context
4. Make sure to toggle "Include LogQL pipeline operations" in the collapsable panel to ON.
5. Previous to this PR you wouldn't see any log lines. With this PR you should see log lines as usual (containing "test" as log lines everywhere)
